src/olympia/amo/tests/test_pagination.py

```diff
@@ -138,10 +138,21 @@ def test_paginate_returns_this_paginator(self):
         pager = paginate(request, qs)
         assert isinstance(pager.paginator, ESPaginator)
 
+
+class TestNonDSLMode(ESTestCase):
+
     def test_count_non_dsl_mode(self):
+        addon_factory()
+        addon_factory()
+        addon_factory()
+
+        self.refresh()
+
         p = ESPaginator(Addon.search(), 20, use_elasticsearch_dsl=False)
 
-        assert p._count is None
+        assert p.count == 3
 
         p.page(1)
+
+        assert p.count == 3
```

Comment:
This is related to earlier pagination changes and is more related to django 1.11 cleanups